verify-merge.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python3
+import argparse
+import os
+import subprocess
+import glob
+
+GIT = os.getenv('GIT','git')
+GPG = os.getenv('GPG','gpg')
+
+def verify():
+    global args, workdir
+    os.chdir('gitian-pubkeys')
+    print('Importing pubkeys...')
+    keys = [f for f in glob.glob("*.asc", recursive=True)]
+    for key in keys:
+        subprocess.check_call([GPG, '--import', key])
+    print('Refreshing pubkeys...')
+    subprocess.check_call([GPG, '--refresh'])
+    os.chdir('../../gitian-builder')
+    print('\nVerifying '+args.version+' Linux\n')
+    subprocess.check_call(['bin/gverify', '-v', '-d', '../gitian.sigs/', '-r', args.version+'-linux', '../monero/contrib/gitian/gitian-linux.yml'])
+    print('\nVerifying '+args.version+' Windows\n')
+    subprocess.check_call(['bin/gverify', '-v', '-d', '../gitian.sigs/', '-r', args.version+'-win', '../monero/contrib/gitian/gitian-win.yml'])
+    print('\nVerifying '+args.version+' MacOS\n')
+    subprocess.check_call(['bin/gverify', '-v', '-d', '../gitian.sigs/', '-r', args.version+'-osx', '../monero/contrib/gitian/gitian-osx.yml'])
```

Comment:
If the user does not build / sign all platforms (many seem to leave out osx) -- this will break.
```python
Verifying v0.14.1.1 MacOS

bin/gverify:82:in `<main>': ../gitian.sigs/v0.14.1.1-osx does not exist (RuntimeError)
Traceback (most recent call last):
  File "./verify-merge.py", line 63, in <module>
    main()
  File "./verify-merge.py", line 59, in main
    verify()
  File "./verify-merge.py", line 22, in verify
    subprocess.check_call(['bin/gverify', '-v', '-d', '../gitian.sigs/', '-r', args.version+'-osx', '../monero/contrib/gitian/gitian-osx.yml'])
  File "/usr/local/Cellar/python/3.7.0/Frameworks/Python.framework/Versions/3.7/lib/python3.7/subprocess.py", line 328, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['bin/gverify', '-v', '-d', '../gitian.sigs/', '-r', 'v0.14.1.1-osx', '../monero/contrib/gitian/gitian-osx.yml']' returned non-zero exit status 1.
```